WordPress/Jetpack/Resources/AppStoreStrings.po

```diff
@@ -38,15 +38,19 @@ msgctxt "app_store_keywords"
 msgid "social,notes,jetpack,writing,geotagging,media,blog,website,blogging,journal"
 msgstr ""
 
-msgctxt "v19.6-whats-new"
+msgctxt "v19.7-whats-new"
 msgid ""
-"Big news, site fans—your My Site screen just got better. You’ll now see two new tabs you can switch between: Site Menu and Home.\n"
+"We made a small but mighty change to the block editor: the “Add Block” button is more visible when you first open the editor with no blocks selected. We also removed one of three error notifications when a media or audio block fails to upload—it felt like overkill.\n"
 "\n"
-"The Home tab also contains handy cards with more information about your site, including drafts, scheduled posts, and today’s stats. Or, if you’re in a hurry, you can easily head over to another section of the app.\n"
+"You’ll hear some accessibility tweaks in the VoiceOver experience when you’re rearranging menu items. Instructions and notices are clearer, and the menu hierarchy makes more sense out loud.\n"
 "\n"
-"We’re all about clear communication. That’s why we’ve added new error text to the photo upload process, just in case you try to upload a photo from your device without giving the app photo permission first.\n"
+"We added a new screen to the site creation process where you can enter your site’s intent. We tested this screen with a small group of users, and we think you’ll like it, too.\n"
 "\n"
-"Finally, when in-app notices pop up, they now fade out more smoothly when you don’t need them anymore. Exit stage left.\n"
+"Web previews won’t cut off bottom-of-the-screen notifications when your browser toolbar is visible.\n"
+"\n"
+"What’s in a name? Well, if your site has one, you’ll see it in the My Site navigation title.\n"
+"\n"
+"When you swipe left on a comment in your Notifications, you won’t see the “Trash” option anymore. Instead you’ll see “Unapprove Comment” and “Approve Comment.” We approve.\n"
```

Comment:
👍  I too feel this wording is safer than the more spicy 🌶️  one Grace propose 😅  (While it was funny, the alternative spicier copy might have been a lost reference that not everybody would have got, and could also have been negatively interpreted)